system/jlib/jsecrets.cpp

```diff
@@ -797,10 +797,70 @@ IPropertyTree *queryMtlsSecretInfo(const char *name)
             if (ca)
                 ca->setProp("@path", filepath.str());
         }
+        // TLS TODO: do we want to always require verify, even if no ca ?
```

Comment:
Enabling verify should make a server require a client certificate.  For our public interfaces we usually won't want that (although we could have a roxie for example that requires client certificates).  For peer to peer mtls we will always want verify on.

I'm not positive what happens when there is no CA.  It may mean, "use the well known global CA list" or it may never work.  SecureSocket has supported that combination for a while though... so worth exploring in another JIRA.

Reply:
ok, thanks.  We can look at that issue separately in the future.